### ISSUE TITLE
PC in the VM opcode trace

### ISSUE THREAD

**Issue by pieterhartel** [trufflesuite/ganache-core#401]
Would it be possible to include the PC in the VM opcode trace? This would make it easier to debug transactions where the source code is unavailable. 

## Expected Behavior
0:PUSH1
2:PUSH1
4:MSTORE 0x60 0x40
5:PUSH1
7:CALLDATASIZE

## Current Behavior
PUSH1
PUSH1
MSTORE 0x60 0x40
PUSH1
CALLDATASIZE

## Possible Solution
In `lib/blockchain_double.js `replace `info.opcode.name` by `info.pc + ':' + info.opcode.name`.

## Your Environment
$ ganache-cli --version
Ganache CLI v6.2.5 (ganache-core: 2.3.3)

**Comment by davidmurdoch** [comment]
To clarify, you want the op's _log_ output to include the `pc`, not the output of the trace itself (which already contains `pc`: https://github.com/trufflesuite/ganache-core/blob/fe5ed79e8f47fce12dd9019bdfe764cb2cf95ab8/lib/blockchain_double.js#L858)

**Comment by pieterhartel** [comment]
@davidmurdoch yes please in the log output, as I wrote under Expected behaviour.

**Comment by davidmurdoch** [comment]
Would logging the results `debug_traceTransaction` in your code not work for you? I'm _slightly_ hesitant to change the log output here as it's unclear if anyone uses this log output in an automated manner; I may need to treat this change as a breaking change (with a major version bump across all affected products and all the fanfare that goes with that).

**Comment by pieterhartel** [comment]
To use `debug_traceTransaction` one would have to change the test script.

It would be more convenient when something goes wrong to fire up `ganache-cli` with appropriate options, for example `ganache-cli --debug --verbose` for slightly more output, and `ganache-cli --debug --very-verbose` for lots of output. This should not be a breaking change.

**Comment by davidmurdoch** [comment]
Ah, ok. I assumed you meant this output was in response to running `debug_traceTransaction`. I see where you want it now. I'll see what we can do. Thanks!
